examples/db-templates/mysql-persistent-template.json

```diff
@@ -4,8 +4,8 @@
   "metadata": {
     "name": "mysql-persistent",
     "annotations": {
-      "openshift.io/display-name": "MySQL",
-      "description": "MySQL database service, with persistent storage. For more information about using this template, including OpenShift considerations, see https://github.com/sclorg/mysql-container/blob/master/5.7/README.md.\n\nNOTE: Scaling to more than one replica is not supported. You must have persistent volumes available in your cluster to use this template.",
+      "openshift.io/display-name": "MySQL (Persistent)",
```

Comment:
I seem to recall we intentionally removed the "Persistent" string because we only install one template in online and it's this one and we didn't want this to show up.  @sspeiche or @jwforres can you recall if that's correct?

Reply:
NP, we can drop it too. It was just copied from `openshift/origin` and never backported.

Reply:
what do you mean copied from openshift/origin?

Reply:
AFAIK the templates 'upstream' moved to their respective container-image repositories.

F.e. this is the 'snapshot' for mysql:
https://github.com/sclorg/mysql-container/commit/a4329802df1ec7032eb4ed8733f4b233e0a7d224

I think it would be best if further changes were made in container repositories first.

Reply:
Sure.  Again the long term goal would be for the templates to live in whatever upstream repository, then be pulled into openshift/library, and then origin would pull from openshift/library.

Reply:
Just confirming, yes we wanted to remove the (Persistent) text from all of these

Reply:
Yes, (Persistent) had been removed. Looking at https://github.com/openshift/origin/blob/master/examples/db-templates/mongodb-persistent-template.json#L7 it isn't there